Assets/scripts/Managers/GameManager.cs

```diff
@@ -32,13 +36,26 @@ public class GameManager : MonoBehaviour {
 	void OnEnable()
 	{
 		SceneManager.sceneLoaded += OnLevelFinishedLoading;
+        ValidateGameConfiguration();
 	}
 
 	void OnDisable()
 	{
 		SceneManager.sceneLoaded -= OnLevelFinishedLoading;
 	}
 
+    void ValidateGameConfiguration()
+    {
+        if (Occuptions.Where(o => o.GetComponent<OccupationRoster>().Type == JobType.ASSISTANT).ToList<GameObject>().Count == 0)
+        {
+            Debug.LogError("Refusing to start server, ASSISTANT role must be defined in the GameManager Occupations to start");
+            #if UNITY_EDITOR
+                        UnityEditor.EditorApplication.isPlaying = false;
+            #endif
+            Application.Quit();
```

Comment:
This check seems like it should be in an [OnValidate()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnValidate.html) handler and call `Debug.LogError` instead of killing the app.